client/src/app/search-result-details/search-result-details.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component,Input } from '@angular/core';
+
+@Component({
+  selector: 'op-search-result-detail',
+  templateUrl: 'app/search-result-detail/search-result-detail.component.html',
+  styleUrls: [],
+  providers: [],
+  directives: [],
+  pipes: [],
+  inputs : ['serachResult','searchResultLebal']
```

Comment:
You don't need to declare the `input` here if you use the `@` way (the `@` is the better way, by angular style guide), so delete this line.